internal/incident/incident.go

```diff
@@ -550,6 +563,31 @@ type RecipientState struct {
 	Role ContactRole
 }
 
+type recipientsChannel map[*recipient.Contact]map[int64]struct{}
```

Comment:
A comment what it's supposed to store would be helpful (contacts, each one with a set of channel IDs). And this should probably also be reflected in the name: it stores contacts, not recipients (that's the more generic term for contact or group or schedule) and also multiple channels. So maybe something like `contactsWithChannels`.